crypto/secp256k1/secp256_test.go

```diff
@@ -129,17 +129,12 @@ func signAndRecoverWithRandomMessages(t *testing.T, keys func() ([]byte, []byte)
 }
 
 func TestRecoveryOfRandomSignature(t *testing.T) {
-	pubkey1, seckey := GenerateKeyPair()
+	pubkey1, _ := GenerateKeyPair()
 	msg := randentropy.GetEntropyCSPRNG(32)
-	sig, err := Sign(msg, seckey)
-	if err != nil {
-		t.Errorf("signature error: %s", err)
-	}
```

Comment:
Not sure what TBH. There are other tests for Sign, this one is supposed to check recovery.